GiEnJul/ClientApp/src/components/admin/connections/completed/DeliveryExcelDownload.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button } from "@material-ui/core";
+import GetAppIcon from "@material-ui/icons/GetApp";
+import React, { useState } from "react";
+
+const DeliveryExcelDownload = () => {
+  const [location, _] = useState("Nittedal");
+  return (
+    <Button
+      variant="contained"
+      color="primary"
+      startIcon={<GetAppIcon />}
+      href={`/api/admin/excel/delivery/${location}`}
```

Comment:
Two things to keep in mind here,
1. this url be different if we split frontend and backend
2. this endpoint should be require authorization.

Could button use an onclick action that makes use of the ApiService class?

Reply:
Changed this into a onclick action that uses the ApiService class. The access token will be passed down later from the AdminPage macro.